pkg/plugins/golang/v4/init.go

```diff
@@ -101,7 +101,9 @@ func (p *initSubcommand) InjectConfig(c config.Config) error {
 	if p.repo == "" {
 		repoPath, err := golang.FindCurrentRepo()
 		if err != nil {
-			return fmt.Errorf("error finding current repository: %w", err)
+			return fmt.Errorf("unable to determine module path: %w. "+
+				"Please use --repo flag to specify your module path "+
+				"(e.g., --repo github.com/user/project)", err)
```

Comment:
This change appears unrelated to the issue scenario. Can we please submit it as a separate PR to keep history and release notes accurate?